src/platform/windows/startup-issues.md

```diff
@@ -65,21 +65,20 @@ If the update process leaves you with multiple Anki installs (such as within
 `C:\Program Files\Anki` and `C:\Program Files (x86)\Anki`), they may be left in a
 non-working state, and Anki may refuse to start without showing an error message.
 
-Try uninstalling all copies - you may be able to do so with the Windows `Apps & features` settings menu, or by running `uninstall.exe` in each Anki program
+Try uninstalling all copies - from Windows Settings > Apps & features (or Apps > Installed apps), or by running `uninstall.exe` in each Anki program
```

Comment:
Generally we wrap UI elements in quotes (unlike most modern styles where bold is used). Is this better?

```suggestion
Try uninstalling all copies of Anki from your computer. To do this, Go to "Settings" > "Apps & features" (or "Apps" > "Installed apps"), or run `uninstall.exe` in each Anki program
```

Reply:
Oof, that's too many quotation marks -- I can't do it! 😅 Is that the house style? I haven't noticed such an abundance of quotes. 

Updated in part

Reply:
I can understand. The house style here isn't consistent. I would've expected UI elements to be quoted in most instances but many a times we don't (arbitrarily). And in this situation it certainly will look cluttered. Google/MS/IBM etc. bold UI element names in their docs. I'm fine either way though, just should be consistent.